### PR TITLE
Improve systray VPN button initialization [OLD]

### PR DESCRIPTION
**Pull request by mrpalide**
Did you run `make format && make check`? Yes

Fixes # _

 Changes:	
- add `IsDMSGClientReady` method to visor for checking availability of dmsg-client [`dmsgC`]. we use `setupPK` to check routes, and `setupPK` use dmsg actually, then need `dmsgC` be ready to use it. If `dmsgC` not ready then VPN button on systray not available. It will check each 10 seconds.
- add retrying logic to fetch vpn-servers from SD

How to test this PR:
- Clone PRs branch
- add `time.Sleep(20 * time.Second)` in line 296 of `init.go` in `pkg > visor > init.go`. Should be like :
  ```
  go func() {
	defer wg.Done()
	time.Sleep(30 * time.Second)
	dmsgC.Serve(context.Background())
  }()
  ```
- `make build-systray` and run visor
- check logs

### CODE REVIEW

File: internal/gui/gui.go [hunk 1]
```diff
@@ -182,19 +182,30 @@ func initVpnClientBtn(conf *visorconfig.V1, httpClient *http.Client, logger *log
...
 	// VPN Public Servers List
-	mVPNServersList := mVPNClient.AddSubMenuItem("Servers", "VPN Client Servers")
-	mVPNServers := []*systray.MenuItem{}
-	for _, server := range getAvailPublicVPNServers(conf, httpClient, logger.PackageLogger("systray:servers")) {
-		mVPNServers = append(mVPNServers, mVPNServersList.AddSubMenuItemCheckbox(server, "", false))
+	go serversBtn(conf, rpcC, httpClient, logger.PackageLogger("systray:vpn-servers"))
+}
+
+func mVPNClientEnable(rpcC visor.API, logger *logging.Logger) {
+	logger.Warn("Initializing...")
```
Comment (ersonp) [inline internal/gui/gui.go]: This should be info or debug.

File: internal/gui/gui.go [hunk 2]
```diff
@@ -182,19 +182,30 @@ func initVpnClientBtn(conf *visorconfig.V1, httpClient *http.Client, logger *log
...
+	for {
+		isReady, _ := rpcC.IsDMSGClientReady()
+		if isReady {
+			mVPNClient.Enable()
+			logger.Info("Initialized.")
+			return
+		}
+		logger.Warn("DSMG-Client not ready yet, will check in next 10 seconds.")
```
Comment (ersonp) [inline internal/gui/gui.go]: `DSMG-Client` is misspelled. `DMSG-Client`

File: internal/gui/gui.go (hunk 1 above)
Comment (mrpalide) [reply internal/gui/gui.go]: Fixed.

File: internal/gui/gui.go (hunk 2 above)
Comment (mrpalide) [reply internal/gui/gui.go]: Fixed.

**Review by ersonp — APPROVED** [review]

**Pull request closed without merging**